tests/capture-tests.js

```diff
@@ -243,26 +265,46 @@ require(["mobifyjs/utils", "capture"], function(Utils, Capture) {
         $("#qunit-fixture").append($iframe);
     });
 
+    // Ensure that a meta viewport tag is added to the document if one is not present
```

Comment:
Won't we be always adding it to the document regardless of whether it's present? I don't see us doing any conditional checks for this.